README.md

```diff
@@ -1,7 +1,7 @@
 Project Ramda
 =============
 
-A practical functional library for Javascript programmers.
+Ramda is the simpler & more practical functional library for Javascript.
```

Comment:
Ok as you wish :smile: 
Though from what i read, you guys are driven by philosophical discussions of idealism, purity, beauty in all your discussions. Any time a user requests a "practical" feature, you counter with philosophy. So you guys are really the steve jobs macintosh and not the bill gates practical windows machines. But if you want practical ok then it is practical :smile: